blocks/page_list/templates/blog_list/view.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+defined('C5_EXECUTE') or die('Access Denied.');
+
+use Concrete\Block\PageList\Controller;
+use Concrete\Core\Localization\Service\Date;
+use Concrete\Core\Page\Page;
+use Concrete\Core\Support\Facade\Application;
+use Concrete\Core\Utility\Service\Text;
+
+/** @var string $noResultsMessage */
+/** @var string $buttonLinkText */
+/** @var Page[] $pages */
+/** @var bool $displayThumbnail */
+/** @var Controller $controller */
+$c = Page::getCurrentPage();
+
+$app = Application::getFacadeApplication();
+/** @var Text $th */
+$th = $th = $app->make(Text::class);
+/** @var Date $dh */
+$dh = $app->make(Date::class);
+
+$includeEntryText = (isset($includeName) && $includeName) || (isset($includeDescription) && $includeDescription) || (isset($useButtonForLink) && $useButtonForLink);
```

Comment:
We are running PHP 7.4 everywhere, you could do
```
$includeEntryText = ($includeName ?? false) || ($includeDescription ?? false) || ($useButtonForLink ?? false);
```
which is a bit easier to read.